### PR TITLE
Hotplug ignore rbd

### PR DESCRIPTION
**Pull request by noahdesu**
// known CI issues
[skip ci]

### CODE REVIEW

File: pkg/daemon/discover/discover.go [hunk 1]
```diff
@@ -167,7 +167,9 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 	// return any add or remove events, but none that match device mapper
 	// events. string matching is case-insensitve
 	events := make(chan string)
-	go rawUdevBlockMonitor(events, []string{"(?i)add", "(?i)remove"}, []string{"(?i)dm-[0-9]+"})
+	go rawUdevBlockMonitor(events,
+		[]string{"(?i)add", "(?i)remove"},
+		[]string{"(?i)dm-[0-9]+", "(?i)rbd[0-9]+"})
```
Comment (galexrt) [inline pkg/daemon/discover/discover.go]: Could this be made configurable by an environment variable?

There are probably some users having other block devices that should not be picked up (like rbd).

Comment (noahdesu) [reply pkg/daemon/discover/discover.go]: @galexrt yes, and we really should. though, there are some questions related to what form those options should take, and resolving that is going to take a bit more time.

File: cluster/examples/kubernetes/ceph/operator.yaml [hunk 2]
```diff
@@ -113,6 +113,9 @@ spec:
         # For more details see https://github.com/rook/rook/issues/2254
         - name: ROOK_ENABLE_FSGROUP
           value: "true"
+        # Disable automatic orchestration when new devices are discovered
+        # - name: ROOK_DISABLE_DEVICE_HOTPLUG
```
[the remark this answers is not in the document]
Comment (noahdesu) [reply cluster/examples/kubernetes/ceph/operator.yaml]: @travisn and that default goes in `values.yaml.tmpl` ?

[the remark this answers is not in the document]
Comment (travisn) [reply cluster/examples/kubernetes/ceph/operator.yaml]: correct

**Comment by galexrt** [comment]
@noahdesu Could you please create a ticket for the filter configuration part?

File: pkg/daemon/discover/discover.go (hunk 1 above)
Comment (nizamial09) [reply pkg/daemon/discover/discover.go]: @noahdesu @galexrt Hey. I've been looking at this issue and would want to work on this. So can you elaborate on this environment variable and how can udevfilter be configured with an environment variable.

**Pull request merged**